templates/engage/fr/esm.html

```diff
@@ -0,0 +1,119 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}Canonical annonce l’Extension Sécurité et Maintenance pour Ubuntu 14.04.{% endblock %}
+{% block meta_description %}Sécurité, conformité, RGPD: gardez vos serveurs en sécurité après la fin de support officielle d'Ubuntu 14.04 LTS en avril 2019.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/14owLexmd5yn9g9Opq2oYAw-oxG2k2YKLgEi9bGqzzMg/edit{% endblock meta_copydoc %}
+
+
+{% block content %}
+
+<section class="p-strip is-deep p-takeover--private-cloud-economics">
```

Comment:
can you add lang="fr" for the containers in french?

Reply:
Very good point, also added to the other fr landing page